packages/modal-ui/src/lib/components/styles.css

```diff
@@ -278,6 +278,7 @@
 }
 
 .nws-modal-wrapper .modal .wallet-options-wrapper .options-list li div {
```

Comment:
Maybe we should consider greying out deprecated wallets any maybe modifying the sorting so they're at the end of the list?